catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -165,13 +165,31 @@ class MDCNodeListViewController: CBCNodeListViewController {
       return
     }
     applyColorScheme(colorScheme)
+    applyThemeOnCurrentExample()
   }
 
   private func applyColorScheme(_ colorScheme: MDCColorScheming) {
     MDCAppBarColorThemer.applySemanticColorScheme(colorScheme, to: appBar)
 
     appBar.navigationBar.tintColor = UIColor.white
   }
+
+  func applyThemeOnCurrentExample() {
+    // This is a short term solution of applying the theme instantly on the
+    // currently presented example. A better solution would be to re-theme the example's components
+    // and not reload the example entirely with a pop-push.
+    guard let topVC = self.navigationController?.topViewController,
+      topVC is MDCAppBarContainerViewController ||
+        topVC.self.responds(to: NSSelectorFromString("catalogBreadcrumbs")) else {
+          return
+    }
+    guard self.navigationController!.popViewController(animated: false) != nil,
```

Comment:
Try to keep one statement per guard for legibility's-sake.

E.g.:

```swift
guard let navigationController = self.navigationController else {
  return
}
```

Reply:
done